drivers/tee/optee/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Epam Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config OPTEE
+	bool "OP-TEE driver"
+	depends on TEE
+	help
+	  This implements suppost of the OP-TEE firmware which is loading
+	  as BL32 image. OP-TEE is a Trust Zone OS which implements mechanisms
```

Comment:
Hm. Maybe. I don't know how how to rephrase it. Let's leave as is until be get support for another architectures?

Reply:
you may do it as it has be done for Linux Kernel: https://elixir.bootlin.com/linux/latest/source/drivers/tee/optee/Kconfig#L8

as for me, it is a really simple description

Reply:
Looked into the description you provided And couldn't understand how it can be applied here. And could you please give me more details about your question "what if op-tee os is used for secure monitor too, it is possible for ARMv7 and for ARMv8 EL3 in AArch32". How it conflicts with already provided help message?

Reply:
The BLxx comment refers to the exact stage of binary load. As far as I know, when using OP-TEE OS as the Secure Monitor, we don't use TF-A, and thus the loading stages will be different. This is possible only in the case of using ARMv7 and ARMv8 ARCH32.

I proposed to avoid mentioning "boot level stages" (BLxx) in the description and instead use a similar description to that used in the Linux Kernel. Moreover, this will reduce conflicts in the future if someone adds support for RISC-V, for example. The term "BL" is commonly used in the context of ARM architecture, but I have seen it used in some presentations referring to RISC-V. But, it may have another meaning or usage for other platforms.

However, the driver is currently used only for ARM64 after the latest changes, so this conversation is likely meaningless...